model_utils.py

```diff
@@ -11,7 +16,12 @@
 import json
 import argparse
 
-from data_utils import *
+from   data_utils  import *
+from   scone_utils import *   # RK - should merge with data_utils ?
```

Comment:
probably don't need this line

Reply:
If you want to remove this, please test it to be sure.